### PR TITLE
(WIP) Initial Aeon Rebase

### PR DESCRIPTION
**Pull request by mfa007**
This updates the cryptonote configs w/ the aeon networks and adds some changes for potential testnet.  Updates the initial checkpoints to aeon current.  This has not been tested, nor have the tests been updated for the new config.  Putting this out there as a POC for initial discussion.  

What wasn't ported:
charity mining
aeons RTC implementation
most everything else that I figured might have its own imp in the upstream monero codebase.

Cheers!

mFa

### CODE REVIEW

**Comment by drparadoxical** [comment]
Ha. "Update by 1?!?!" 

I thought the same damn thing. Found out that'll come in handy later if and when a testnet for Aeon is in place, that's why it's even a configuration variable found with the XMR code base to begin with. 

Thanks for your initiative, let me go through this pull request tomorrow before merging, haven't yet established our starting point, but don't mind at all with this pull being it. 

Back with you again tomorrow.

**Comment by devopsralf** [comment]
It's happening! Testnet would be awesome to have.

**Comment by devopsralf** [comment]
So @mfa007 you changes the dockerfile to reference https://github.com/aeon-project/aeon.git
I saw that this was not taken yet and took the liberty to register that organization before anyone else sniped it. If we move forward and want to use that, we have it secured.

**Comment by drparadoxical** [comment]
@mfa007 - did you clone the Monero 0.11.1.0 tag when you started, or something else? We're over in task/5 talking about our starting point, and we'll figure that out before determining what to do with your PR here. 

And thank you, good work, love the initiative, nothing worse than that cold water when first diving in, but I see that doesn't bother you much at all. Good on you sir.

**Comment by mfa007** [comment]
I did as we had mentioned and downloaded the latest monero release zip, not master.  I also went with the zip in order to sidestream all the potential git conflicts and get to progressing.  

Whatever you think on the PR, I've actually progressed quite a bit past that and have the nodes on the aeon network with some hiccups.  I've worked on more dev teams than I can count and I'm not about to let this bounty go to someone else bc someone feels they need to lecture me on how to git or instantly update readmes and licenses.The whole point of bounty is fast iteration and get it done w cleanup and formalities at least after something is working.

Sure if we were setting up for the long haul and plan on maintaining the project beyond the bounty.  But that is not my intentions and I can not get caught up in some amateur  donkey show contest with another dev who thinks the problems can be solved by throwing countless libraries at an issue.  I have handled projects on teams as well as by self and if on a team, I would expect not to have to get caught up in any burecratic BS and let the people who get things done get things done.  

I'm a hardware and software guy and feel just fine doing architecture, engineering and development but as far as managing goes, most good managers that I've had let the talent grow and wouldn't get much out of me asking for specific features to their spec.  See a problem and then design, engineer and implement a solution.  

This stuff seems all pretty simple to me and I feel I'm more than qualified.  Heck, I've even got a crypto sec patent behind me.  What I'm trying to get at, if you want to hold me back with formal, bs and/or feel clueless to think  you'd need me to learn how to use some basic tooling, I'm not the right guy.  If your feeling up for helping finish this poc I have running now, please feel free to help and even provide some feedback and contributions.  But don't expect me to follow your or anyone's for that matter every demand.  

Don't mean for this to rant and don't feel as if this is directly targeting you, bc it's not.  You've been straight up and cool since we've met.  I don't have time to compare development environments or privates sizes 

I'm usually always tied up Friday evenings with prior engagements until 2am but can hop on after that.  Going to look at some cheap vps options to get testnet seeds running to help test out the implementation bits but so far, seems to connect and do all the right things.

Take care and we'll talk later.

mFa

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2014-2017, The Monero Project
```
Comment (guzzijones) [inline CMakeLists.txt]: Not sure this is ok.

**Comment by drparadoxical** [comment]
Good, and again, I appreciate you taking the bull by the horns. In all honesty, I'd have copied the repo in the same way, downloading a package and starting there, so you shouldn't be thinking you've made a mistake of some sort or done anything wrong. 

I understand you perfectly in your intent and attempt for quick and dirty. I'll never argue with anyone who's all about just getting to the damn point. The contention here is accidental, and concerns the matter of us not removing copyright information, disclaimers, or going against the license set forth by the project we're forking. 

So let's just clear the air now - because I don't think for a second you're some deviant who is intentionally stripping copyright disclaimers from the headers of files in order to, I don't know, whatever. Who cares. I do think that's all some of the guys are trying to say, not that you've done fucked up and how dare you. 

Bureaucratic bullshit nonsense, sure, I doubt any of us can reasonably argue otherwise, so let's move past the bullshit nonsense. 

Are you set on not including the license and copyright, is it a deal breaker, are you absolutely unwillingly to work with the team unless that copyright's removed? 

If not, then problem solved, case closed, and your contributions already are welcomed and very much appreciated. 

Now gentlemen, either throw some fists outside and come back in when you're done, or close in and shut down the single, small point of contention I don't believe any of us actually disagree on. 

Your talents and time are wasted if you don't shutter this now, though. Too much strength here to just let a start up mishap and misunderstanding get in the way of what we're about to do together, and it'll be a damn shame to lose any of you before I even had the chance to greet you properly and welcome everyone to the team.

File: CMakeLists.txt (hunk 1 above)
Comment (drparadoxical) [reply CMakeLists.txt]: Just commented on this, and no, it's not OK, but I also think it's not something that was intentionally done.

**Comment by drparadoxical** [comment]
Speaking to your technical points @mfa007, there's a GCP account created specifically for this work, with an operating budget set aside to pay for whatever service or VM or API is spun up or used, and a node can easily run on a compute cluster VM, AFAIK. Not saying this is what has to be done, but want to make sure everyone here knows no one has to spend their own cash when it comes to this work - if something is needed, operating budget will handle it, and I certainly don't expect to own, manage, or even have access to, whatever it is.

Greasing the skids, is all, and wanted to make sure you knew before diving into the VPS related aspect of your comment and work you mentioned.

**Comment by mfa007** [comment]
Looks like there is quite the bunch of contributors.  I've cleaned up this PR and moved my changes into a private repo.  Can tell you that based on what I'm seeing via upnp discovery, there appears to be others that have gotten a fork onto the aeonnet.  

I cannot be on a team that nits little things and the way I see it, if you see something that you can fix while other things might be more advanced, doing those changes yourself is collaboration, not telling someone how they need to conform to your guidelines when the project is not even released and may well never be...Best of luck.

**Comment by drparadoxical** [comment]
No harm, no foul. We'll just compare notes on the other side of this thing. 

My fear is that you see yourself outside of this, when the truth is, we're never really off this same team. 

As I said in the neighboring thread meant to manage the bulk of discussion for a spell, I expected a settling period to be needed when it comes to these types of things, and even as you exit, do so knowing  you weren't asked to leave, and that your contributions are always welcome back. 

For perseverance and reference sake, let it be known, that you took the initiative to initiate the rebase, and in doing so, just happened to do it in a way that, accidentally or otherwise, stripped copyright headers from files. That was the entirety of it. 

It's never too late to rejoin. If worried about reputation or being judged, I can tell you first hand, no one other than yourself has true control over these things. 

And yes, I know, others have joined the fray, mad dash to cash and all. I care you not. 

You know nothing of what I can and will do. 

I was just sure hoping you'd be there for it.

**Pull request closed without merging**